Jenkinsfile

```
@@ -140,6 +140,14 @@ pipeline {
 				sh 'pm2 stop --silent ecosystem.jenkins.config.js'
 			}
 
+			dir('./') { sh 'rm -rf node_modules' }
+			dir('./framework') { sh 'rm -rf node_modules' }
+			dir('./services/core') { sh 'rm -rf node_modules' }
+			dir('./services/market') { sh 'rm -rf node_modules' }
+			dir('./services/gateway') { sh 'rm -rf node_modules' }
+			dir('./services/template') { sh 'rm -rf node_modules' }
+			dir('./tests') { sh "rm -rf node_modules" }
+
 			dir('./jenkins/lisk-core') { sh "make down" }
 			dir('./jenkins/mysql') { sh "make down" }
 			dir('./jenkins/redis') { sh "make down" }
```

Kindly consider performing the cleanup after all the processes are down.
```suggestion
			dir('./jenkins/lisk-core') { sh "make down" }
			dir('./jenkins/mysql') { sh "make down" }
			dir('./jenkins/redis') { sh "make down" }
			
			dir('./') { sh 'rm -rf node_modules' }
			dir('./framework') { sh 'rm -rf node_modules' }
			dir('./services/core') { sh 'rm -rf node_modules' }
			dir('./services/market') { sh 'rm -rf node_modules' }
			dir('./services/gateway') { sh 'rm -rf node_modules' }
			dir('./services/template') { sh 'rm -rf node_modules' }
			dir('./tests') { sh "rm -rf node_modules" }
```